app/assets/javascripts/discourse/app/controllers/tag-show.js

```diff
@@ -67,11 +67,6 @@ export default Controller.extend(BulkTopicSelection, FilterModeMixin, {
     });
   },
 
-  @discourseComputed("category")
-  showTagFilter() {
```

Comment:
It looks like a couple custom themes use this computed property so I don't think it's safe to remove.